aws/tf/modules/sra/databricks_account/uc_assignment/uc_assignment.tf

```diff
@@ -1,6 +1,11 @@
 // Metastore Assignment
+
+data "databricks_metastore" "this" {
```

Comment:
Yeah I thought about this on implementation - since this isn't a normal situation, I thought of leaving it and making a comment about it - since I did remove the metastore ID as a variable in general

Reply:
Fair point.  I imagine the number of folks encountering this scenario to be quite small.  A comment would be a nice addition though methinks.